app/uk/gov/hmrc/gform/models/helpers/Javascript.scala

```diff
@@ -91,7 +91,8 @@ object Javascript {
          |  return BigNumber(a).times(BigNumber(b));
          |};
          |function displaySterling(result) {
-         |  return result < 0 ? result.replace("-", "-£") : '£' + result
+         |
+         |  return result < 0 ? result.replace("-", "-£") : '£' BigNumber(result).toFormat(2)
```

Comment:
we are missing the + symbol concatenation, required after the '£' to make a single string - this throws a JS error for unexpected identifier